Makefile

```diff
@@ -166,16 +166,21 @@ ifdef LLAMA_CUBLAS
 	OBJS      += ggml-cuda.o
 	NVCC      = nvcc
 	NVCCFLAGS = --forward-unknown-to-host-compiler -arch=native
+ifdef LLAMA_CUDA_FORCE_DMMV
+	NVCCFLAGS += -DGGML_CUDA_FORCE_DMMV=$(LLAMA_CUDA_FORCE_DMMV)
```

Comment:
This is fine, but the value is not used anyway, for clarity I suggest changing this to just `-DGGML_CUDA_FORCE_DMMV`.